ansible/roles/test/files/ptftests/sflow_test.py

```diff
@@ -209,15 +209,15 @@ def analyze_flow_sample(self, data, collector):
     #---------------------------------------------------------------------------
 
     def sendTraffic(self):
-        self.src_ip_list = ['192.158.8.1','192.168.16.1', '192.168.24.1','192.168.32.1']
+        src_ip_addr_templ = '192.168.{}.1'
         ip_dst_addr = '192.168.0.4'
         src_mac = self.dataplane.get_mac(0, 0)
         pktlen=100
         #send 100 * sampling_rate packets in each interface for better analysis
         for j in range(0, 100, 1):
             index = 0
             for intf in self.interfaces:
-                ip_src_addr = str(self.src_ip_list[index])
+                ip_src_addr = src_ip_addr_templ.format(str(8 * index))
```

Comment:
>8 * index [](http://example.com/codeflow?start=59&length=9)

Will it overflow for ipv4?

Reply:
For t0 topo, number of self.interfaces would be 4 and for t0-56 topo, this would be 8. [Ref](https://github.com/Azure/sonic-mgmt/blob/master/docs/testbed/README.testbed.Topology.md)

So no, this wouldn't overflow for this t0-56 topology, as it runs from `192.168.0.1 to 192.168.56.1`